moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectsTable.js

```diff
@@ -177,6 +177,7 @@ const SubprojectsTable = ({ projectId = null }) => {
             })
               .then(() => {
                 refetch();
+                refetchSummaryData(); // Refresh subprojects in summary map
```

Comment:
We refetch summary data so that the data going to the summary map updates and the map updates.

Reply:
this works great! ✨